core/src/main/java/software/amazon/awssdk/core/interceptor/ExecutionInterceptor.java

```diff
@@ -53,7 +57,7 @@
  * </ol>
  * An additional {@link #onExecutionFailure} method is provided that is invoked if an execution fails at any point during the
  * lifecycle of a request, including exceptions being thrown from this or other interceptors.
- * </p>
+ * <p></p>
```

Comment:
do we need these empty tags `<p></p>` here?  I saw them in other files as well

Reply:
Yes, self-closing tags are not allowed.

Reply:
Gotcha, maybe we can remove the closing tags? from http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html, `</p>` doesn't seem to be needed.

Reply:
Nice, sounds good!